src/styles/global.css

```
@@ -262,8 +262,84 @@ a:focus-visible {
+  user-select: none;
+}
+
+.timeline-filter-link:hover {
+  color: var(--ink);
+  text-decoration-color: color-mix(in srgb, var(--line) 90%, transparent);
+  background: color-mix(in srgb, var(--ink) 4%, transparent);
+}
+
+.timeline-filter-link[aria-pressed="true"] {
+  color: var(--ink);
+  text-decoration-color: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)